### PR TITLE
Add support for raw key events on Fuchsia

### PR DESCRIPTION
**Pull request by abarth**
Fixes https://github.com/flutter/flutter/issues/6631

### CODE REVIEW

**Comment by abarth** [comment]
@j9brown

File: content_handler/runtime_holder.cc [hunk 1]
```diff
@@ -205,6 +208,33 @@ void RuntimeHolder::OnEvent(mozart::EventPtr event,
...
+    const char* type = nullptr;
+    if (event->action == mozart::EventType::KEY_PRESSED)
+      type = "keydown";
+    else if (event->action == mozart::EventType::KEY_RELEASED)
+      type = "keyup";
+
+    if (type) {
+      rapidjson::Document document;
+      auto& allocator = document.GetAllocator();
+      document.AddMember("type", rapidjson::Value(type, strlen(type)),
```
Comment (chinmaygarde) [inline content_handler/runtime_holder.cc]: I am pretty sure you are going to trip an assertion on this first `AddMember`. You have specified that the document root is an object. `document.SetObject();` should do the trick.

Comment (chinmaygarde) [inline content_handler/runtime_holder.cc]: *not specified

Comment (abarth) [inline content_handler/runtime_holder.cc]: Thanks.

**Comment by abarth** [comment]
Done.  PTAL

**Comment by Hixie** [comment]
are we really going to encode and parse an entire JSON blob for each key press? Or am I misreading this?

**Comment by abarth** [comment]
> are we really going to encode and parse an entire JSON blob for each key press?

That's correct.  I doubt it's a measurable overhead.

**Pull request merged**